packages/js-sdk/tests/sandbox/host.test.ts

```diff
@@ -2,30 +2,63 @@ import { assert } from 'vitest'
 
 import { isDebug, sandboxTest, wait } from '../setup.js'
 
-sandboxTest('ping server in sandbox', async ({ sandbox }) => {
-  const cmd = await sandbox.commands.run('python -m http.server 8000', { background: true })
+sandboxTest.skipIf(isDebug)(
```

Comment:
this should work in debug right? maybe you could use nonstandard random port to prevent some port conflict 
```suggestion
sandboxTest(
```